apps/vs-code-designer-react/src/app/DesignerCommandBar/index.tsx

```diff
@@ -85,7 +86,10 @@ export const DesignerCommandBar: React.FC<DesignerCommandBarProps> = ({ isRefres
     disableGrey: [{ color: 'rgb(121, 119, 117)' }, iconClass],
   });
 
-  const allOperationErrors = useSelector((state: RootState) => {
+  const designerIsDirty = useIsDesignerDirty();
+
+  const allOperationErrors = useSelector((state: RootState) => Object.entries(state.operations.errors) ?? []);
```

Comment:
This is where we get the errors from invalid connections, this shouldn't block saving as per Travis' recent message